lib/src/widgets/yust_file_picker.dart

```diff
@@ -299,6 +303,9 @@ class YustFilePickerState extends State<YustFilePicker> {
       allowMultiple: widget.allowMultiple,
     );
     if (result != null) {
+      if (_fileHandler.getFiles().isEmpty) {
+        widget.onInitialUpload?.call();
+      }
```

Comment:
This logic should be in _uploadFile()

Reply:
The function has now been deleted and replaced by an onChanged-call.